example_cpp_smart_card_client_app/src/chrome_certificate_provider/api_bridge.cc

```diff
@@ -61,9 +58,11 @@ void ProcessCertificatesRequest(
       locked_certificates_request_handler = certificates_request_handler.lock();
   GOOGLE_SMART_CARD_CHECK(locked_certificates_request_handler);
   if (locked_certificates_request_handler->HandleRequest(&certificates)) {
-    // TODO(#220): Build `Value` directly, without converting from `pp::Var`.
-    result_callback(gsc::GenericRequestResult::CreateSuccessful(
-        gsc::ConvertPpVarToValueOrDie(gsc::MakeVarArray(certificates))));
+    gsc::Value response(gsc::Value::Type::kArray);
+    response.GetArray().push_back(gsc::MakeUnique<gsc::Value>(
+        gsc::ConvertToValueOrDie(std::move(certificates))));
+    result_callback(
+        gsc::GenericRequestResult::CreateSuccessful(std::move(response)));
```

Comment:
Probably I should introduce a helper that transforms the given sequence of C++ objects into the resulting array `Value`, or even into a `GenericRequestResult`. But I'd prefer to postpone that cleanup till later.